### ISSUE TITLE
Digital Pigeon 4 - SparkleUpdateInfoProvider: Error: Error parsing XML from appcast feed.

### ISSUE THREAD

**Issue by macmule** [autopkg/dataJAR-recipes#149]
Looks to be an issue their side:

![Screenshot 2021-05-14 at 06 40 18](https://user-images.githubusercontent.com/2464974/118279015-8a302c00-b4c2-11eb-8722-e0e15c1b4fb2.png)

Sparkle xml:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <title>Digital Pigeon - Access denied</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="robots" content="noindex">
    
    <meta name="_csrf" content="30b1f581-73e0-4dab-af46-817df0db7368"/>
    <meta name="_csrf_header" content="X-CSRF-TOKEN"/>
    <meta name="_csrf_parameter" content="_csrf"/>
    
    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,700italic,900,400italic,300' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" id="shiftnav-css" href="https://www.digitalpigeon.com/wp-content/plugins/shiftnav-responsive-mobile-menu/assets/css/shiftnav.min.css" type="text/css" media="all">
    <link rel="stylesheet" id="dp-shiftnav-css" href="https://www.digitalpigeon.com/wp-content/themes/digitalpigeon/library/css/shiftnav.css" type="text/css" media="all">
    <link rel="stylesheet" id="dp-superfish-css" href="https://www.digitalpigeon.com/wp-content/themes/digitalpigeon/library/js/libs/superfish/css/superfish.css" type="text/css" media="all">
    <link rel="stylesheet" id="shiftnav-font-awesome-css" href="https://www.digitalpigeon.com/wp-content/plugins/shiftnav-responsive-mobile-menu/assets/css/fontawesome/css/font-awesome.min.css" type="text/css" media="all">
    <style type="text/css">
        html { min-height: 100%; }
body { font-family: sans-serif; }
body.loading-content header, body.loading-content .shiftnav, body.loading-content #digi-dialog-powered-by { /*used by webpack to hide page state before css is dynamically loadedd*/ display: none; }
@-moz-keyframes spin { from { -moz-transform: rotate(0deg); } to { -moz-transform: rotate(360deg); } }
@-webkit-keyframes spin { from { -webkit-transform: rotate(0deg); } to { -webkit-transform: rotate(360deg); } }
@keyframes spin { from { transform: rotate(0deg); } to { transform: rotate(360deg); } }
@-moz-keyframes anti-spin { from { -moz-transform: rotate(360deg); } to { -moz-transform: rotate(0deg); } }
@-webkit-keyframes anti-spin { from { -webkit-transform: rotate(360deg); } to { -webkit-transform: rotate(0deg); } }
@keyframes anti-spin { from { transform: rotate(360deg); } to { transform: rotate(0deg); } }
.loading { font-size: 0px; display: block; text-align: center; font-weight: bold; color: rgba(0, 0, 0, 0.3); margin: 80px; text-indent: -999em; overflow: hidden; width: 50px; height: 50px; }
.loading i:after { display: block; -webkit-animation: spin 1s infinite steps(12); -moz-animation: spin 1s infinite steps(12); animation: spin 1s infinite steps(12); left: 50%; top: 50%; margin-top: -25px; margin-left: -25px; position: absolute; width: 50px; height: 50px; opacity: 0.3; content: url("data:image/svg+xml;utf8,<svg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' viewBox='0 0 512 512' width='50' height='50'><path d='M256 151.548c-16.482 0-29.843-13.362-29.843-29.843v-89.53c0-16.482 13.361-29.843 29.843-29.843s29.844 13.361 29.844 29.843v89.53c0 16.482-13.363 29.843-29.844 29.843z'></path> <path d='M256 498.478c-10.301 0-18.652-8.351-18.652-18.652v-89.53c0-10.301 8.351-18.652 18.652-18.652 10.302 0 18.652 8.352 18.652 18.652v89.53c0 10.301-8.35 18.652-18.652 18.652z'></path><path d='M188.878 167.68c-9.67 0-19.073-5.017-24.256-13.994l-44.765-77.535c-7.726-13.382-3.141-30.493 10.241-38.219 13.381-7.725 30.493-3.142 38.219 10.241l44.765 77.536c7.726 13.382 3.141 30.493-10.241 38.219-4.405 2.542-9.216 3.752-13.963 3.752z'></path> <path d='M367.928 466.628c-5.801 0-11.443-3.011-14.554-8.396l-44.765-77.535c-4.635-8.028-1.885-18.296 6.145-22.932 8.028-4.632 18.295-1.885 22.932 6.146l44.766 77.534c4.635 8.029 1.884 18.296-6.146 22.932-2.643 1.524-5.53 2.251-8.378 2.251z'></path> <path d='M139.672 214.97c-4.43 0-8.919-1.128-13.032-3.503l-77.536-44.765c-12.489-7.211-16.769-23.181-9.558-35.671s23.182-16.769 35.671-9.558l77.535 44.765c12.49 7.211 16.769 23.181 9.558 35.67-4.834 8.378-13.613 13.062-22.638 13.062z'></path> <path d='M449.824 382.837c-2.532 0-5.098-0.645-7.447-2.002l-77.534-44.765c-7.137-4.12-9.582-13.247-5.462-20.384s13.248-9.583 20.383-5.462l77.535 44.766c7.137 4.12 9.582 13.246 5.462 20.383-2.765 4.787-7.781 7.464-12.937 7.464z'></path> <path d='M121.705 280.248h-89.53c-13.392 0-24.248-10.856-24.248-24.248s10.856-24.248 24.248-24.248h89.53c13.392 0 24.248 10.856 24.248 24.248s-10.856 24.248-24.248 24.248z'></path> <path d='M479.825 270.922c-0.001 0 0 0 0 0h-89.53c-8.241-0.001-14.922-6.682-14.922-14.922s6.682-14.922 14.922-14.922c0.001 0 0 0 0 0h89.53c8.241 0 14.922 6.681 14.922 14.922s-6.682 14.922-14.922 14.922z'></path> <path d='M62.183 390.299c-7.736 0-15.259-4.014-19.405-11.195-6.181-10.705-2.513-24.395 8.192-30.574l77.536-44.766c10.705-6.184 24.395-2.514 30.575 8.192 6.181 10.706 2.513 24.395-8.192 30.575l-77.536 44.765c-3.525 2.035-7.374 3.003-11.17 3.003z'></path> <path d='M372.316 203.776c-5.157 0-10.173-2.676-12.937-7.463-4.12-7.137-1.675-16.263 5.462-20.384l77.535-44.764c7.136-4.118 16.263-1.676 20.384 5.461 4.12 7.137 1.675 16.263-5.462 20.384l-77.535 44.764c-2.35 1.357-4.915 2.002-7.447 2.002z'></path> <path d='M144.068 470.358c-3.481 0-7.008-0.887-10.24-2.752-9.813-5.666-13.175-18.214-7.51-28.027l44.765-77.535c5.665-9.814 18.213-13.176 28.027-7.511 9.813 5.666 13.175 18.215 7.51 28.027l-44.765 77.536c-3.799 6.583-10.696 10.262-17.787 10.262z'></path> <path d='M323.133 154.621c-2.531 0-5.098-0.645-7.447-2.001-7.137-4.121-9.582-13.247-5.462-20.383l44.767-77.535c4.12-7.137 13.246-9.583 20.383-5.461s9.582 13.247 5.462 20.383l-44.766 77.535c-2.765 4.785-7.78 7.462-12.937 7.462z'></path> </svg>"); }
.loading.compact { margin: 5px; }
/*loading spinner*/
.cssload-tetrominos { position: absolute; left: 50%; margin-left: 47px; top: 50%; opacity: 0.3; transform: translate(-109px, -94px) scale(0.5); -o-transform: translate(-109px, -94px) scale(0.5); -ms-transform: translate(-109px, -94px) scale(0.5); -webkit-transform: translate(-109px, -94px) scale(0.5); -moz-transform: translate(-109px, -94px) scale(0.5); }
.cssload-tetromino { width: 94px; height: 109px; position: absolute; transition: all ease 0.35s; -o-transition: all ease 0.35s; -ms-transition: all ease 0.35s; -webkit-transition: all ease 0.35s; -moz-transition: all ease 0.35s; background: url('data:image/svg+xml;utf-8,%3Csvg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 612 684"%3E%3Cpath fill="%23010101" d="M305.7 0L0 170.9v342.3L305.7 684 612 513.2V170.9L305.7 0z"/%3E%3Cpath fill="%23fff" d="M305.7 80.1l-233.6 131 233.6 131 234.2-131-234.2-131"/%3E%3C/svg%3E') no-repeat top center; }
.cssload-box1 { animation: cssload-tetromino1 1.23s ease-out infinite; -o-animation: cssload-tetromino1 1.23s ease-out infinite; -ms-animation: cssload-tetromino1 1.23s ease-out infinite; -webkit-animation: cssload-tetromino1 1.23s ease-out infinite; -moz-animation: cssload-tetromino1 1.23s ease-out infinite; }
.cssload-box2 { animation: cssload-tetromino2 1.23s ease-out infinite; -o-animation: cssload-tetromino2 1.23s ease-out infinite; -ms-animation: cssload-tetromino2 1.23s ease-out infinite; -webkit-animation: cssload-tetromino2 1.23s ease-out infinite; -moz-animation: cssload-tetromino2 1.23s ease-out infinite; }
.cssload-box3 { animation: cssload-tetromino3 1.23s ease-out infinite; -o-animation: cssload-tetromino3 1.23s ease-out infinite; -ms-animation: cssload-tetromino3 1.23s ease-out infinite; -webkit-animation: cssload-tetromino3 1.23s ease-out infinite; -moz-animation: cssload-tetromino3 1.23s ease-out infinite; z-index: 2; }
.cssload-box4 { animation: cssload-tetromino4 1.23s ease-out infinite; -o-animation: cssload-tetromino4 1.23s ease-out infinite; -ms-animation: cssload-tetromino4 1.23s ease-out infinite; -webkit-animation: cssload-tetromino4 1.23s ease-out infinite; -moz-animation: cssload-tetromino4 1.23s ease-out infinite; }
@keyframes cssload-tetromino1 { 0%, 40% { transform: translate(0, 0); } 50% { transform: translate(47px, -26px); } 60%, 100% { transform: translate(94px, 0); } }
@-o-keyframes cssload-tetromino1 { 0%, 40% { -o-transform: translate(0, 0); } 50% { -o-transform: translate(47px, -26px); } 60%, 100% { -o-transform: translate(94px, 0); } }
@-ms-keyframes cssload-tetromino1 { 0%, 40% { -ms-transform: translate(0, 0); } 50% { -ms-transform: translate(47px, -26px); } 60%, 100% { -ms-transform: translate(94px, 0); } }
@-webkit-keyframes cssload-tetromino1 { 0%, 40% { -webkit-transform: translate(0, 0); } 50% { -webkit-transform: translate(47px, -26px); } 60%, 100% { -webkit-transform: translate(94px, 0); } }
@-moz-keyframes cssload-tetromino1 { 0%, 40% { -moz-transform: translate(0, 0); } 50% { -moz-transform: translate(47px, -26px); } 60%, 100% { -moz-transform: translate(94px, 0); } }
@keyframes cssload-tetromino2 { 0%, 20% { transform: translate(94px, 0px); } 40%, 100% { transform: translate(140px, 26px); } }
@-o-keyframes cssload-tetromino2 { 0%, 20% { -o-transform: translate(94px, 0px); } 40%, 100% { -o-transform: translate(140px, 26px); } }
@-ms-keyframes cssload-tetromino2 { 0%, 20% { -ms-transform: translate(94px, 0px); } 40%, 100% { -ms-transform: translate(140px, 26px); } }
@-webkit-keyframes cssload-tetromino2 { 0%, 20% { -webkit-transform: translate(94px, 0px); } 40%, 100% { -webkit-transform: translate(140px, 26px); } }
@-moz-keyframes cssload-tetromino2 { 0%, 20% { -moz-transform: translate(94px, 0px); } 40%, 100% { -moz-transform: translate(140px, 26px); } }
@keyframes cssload-tetromino3 { 0% { transform: translate(140px, 26px); } 20%, 60% { transform: translate(94px, 53px); } 90%, 100% { transform: translate(47px, 26px); } }
@-o-keyframes cssload-tetromino3 { 0% { -o-transform: translate(140px, 26px); } 20%, 60% { -o-transform: translate(94px, 53px); } 90%, 100% { -o-transform: translate(47px, 26px); } }
@-ms-keyframes cssload-tetromino3 { 0% { -ms-transform: translate(140px, 26px); } 20%, 60% { -ms-transform: translate(94px, 53px); } 90%, 100% { -ms-transform: translate(47px, 26px); } }
@-webkit-keyframes cssload-tetromino3 { 0% { -webkit-transform: translate(140px, 26px); } 20%, 60% { -webkit-transform: translate(94px, 53px); } 90%, 100% { -webkit-transform: translate(47px, 26px); } }
@-moz-keyframes cssload-tetromino3 { 0% { -moz-transform: translate(140px, 26px); } 20%, 60% { -moz-transform: translate(94px, 53px); } 90%, 100% { -moz-transform: translate(47px, 26px); } }
@keyframes cssload-tetromino4 { 0%, 60% { transform: translate(47px, 26px); } 90%, 100% { transform: translate(0, 0); } }
@-o-keyframes cssload-tetromino4 { 0%, 60% { -o-transform: translate(47px, 26px); } 90%, 100% { -o-transform: translate(0, 0); } }
@-ms-keyframes cssload-tetromino4 { 0%, 60% { -ms-transform: translate(47px, 26px); } 90%, 100% { -ms-transform: translate(0, 0); } }
@-webkit-keyframes cssload-tetromino4 { 0%, 60% { -webkit-transform: translate(47px, 26px); } 90%, 100% { -webkit-transform: translate(0, 0); } }
@-moz-keyframes cssload-tetromino4 { 0%, 60% { -moz-transform: translate(47px, 26px); } 90%, 100% { -moz-transform: translate(0, 0); } }
/*end loading spinner*/


        body {
            margin: 0;
        }

        header, header [class^="col-"], header [class^=" col-"] {
            height: 75px;
            z-index: 1000;
        }

        header {
            background-color: #3A80E1;
            width: 100%;
            display: block;

        }

        header #logo-header {
            top: 21px;
            position: relative;
        }

        header .btn.btn-lg {
            width: 150px;
            padding-top: 13px;
            padding-bottom: 13px;
            margin-left: 0 !important;
        }

        .btn-transparent, .header .btn-transparent {
            border: 1px solid #fff;
            background-color: transparent;
            color: #fff;
        }

        .btn.btn-lg {
            font-size: 14px;
            padding: 15px 30px;
        }

        .align-vertical {
            height: auto;
            position: relative;
            top: 50%;
            transform: translateY(-50%);
        }

        .btn {
            -webkit-border-radius: 3px;
            border-radius: 3px;
            font-weight: 300;
            min-width: 175px;
        }

        .btn-green {
            background-color: #7DC855;
            border: 1px solid #7dc855;
            color: #fff;
        }

        .btn-green:hover {
            background-color: #6EC341;
            border-color: #6EC341;
        }

        .btn-green:hover, .btn-blue:hover {
            color: #fff !important;

        }

        .btn-transparent:hover, .header .btn-transparent:hover {
            border-color: #7DC855;
            color: #7DC855;
        }

        .text-right {
            text-align: right;
        }

        @media (min-width: 1200px) {
            .container {
                width: 1170px;
            }

        }

        @media (min-width: 992px) {
            .container {
                width: 970px;
            }
        }

        @media (min-width: 768px) {
            .container {
                width: 750px;
            }
        }

        .container {
            max-width: auto !important;
            padding-right: 15px !important;
            padding-left: 15px !important;
            margin-right: auto;
            margin-left: auto;
        }

        header.header.banner--clone .mobile-menu-button {
            top: 17px;
        }

        .mobile-menu-button {
            position: absolute;
            right: 15px;
            top: 25px;
            border: 2px solid #fff;
            color: #fff;
            padding: 0 10px;
            font-size: 25px;
            cursor: pointer;
            transition: all 0.5s ease;
        }

        .mobile-menu-button:hover {
            background-color: #7DC855;
            color: #fff;
            border-color: #7DC855;
        }

        header .sf-menu {
            margin-bottom: 0;
            margin-right: 20px;
        }

        header .sf-menu li, header .sf-menu li:hover {
            background: transparent;
        }

        header .sf-menu a, header .btn {
            border-left: 0;
            border-top: 0;
            padding: 5px 14px;
            text-transform: uppercase;
            color: #fff;
        }

        header .sf-menu a:hover, header .btn:hover {
            color: #7dc855;
            text-decoration: none;
        }

        header .btn-login {
            border: none !important;
        }

    </style>
    
    
    <meta name="contentTitle" content="404: Access denied">
    <meta name="robots" content="noindex, nofollow">

</head>

<body class="external loading-content">
<header class="header " role="banner" itemscope="" itemtype="http://schema.org/WPHeader">
    <div class="container">
        <div class="row">
            <div class="col-md-3">
                <a href="https://www.digitalpigeon.com?utm_source=app&utm_medium=app-link" id="logo-header"><img src="https://cdn.digitalpigeon.com/digitalpigeon/assets/v/60bb8ff/assets/img/app-logo.png" style="width: 151px" class="img-responsive"></a>
                <div class="mobile-menu-button hidden-lg hidden-md shiftnav-toggle" data-shiftnav-target="shiftnav-main"><i class="fa fa-bars" aria-hidden="true"></i></div>
            </div>
            <div class="col-md-9 text-right">
                <a href="https://digitalpigeon.com/login" class="btn btn-lg btn-transparent btn-login align-vertical pull-right hidden-sm hidden-xs">Login</a>
            	<a href="https://www.digitalpigeon.com/trial?utm_source=app&utm_medium=app-link" class="btn btn-lg btn-green align-vertical pull-right hidden-sm hidden-xs" style="margin-right:10px;">Start Free Trial</a>
            	<ul id="menu-main-menu" class="sf-menu align-vertical pull-right hidden-sm hidden-xs"><li id="menu-item-625" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-625"><a href="https://www.digitalpigeon.com/features/send-large-files/?utm_source=app&utm_medium=app-link">Features</a></li>
                    <li id="menu-item-47" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-47"><a href="https://www.digitalpigeon.com/customers/?utm_source=app&utm_medium=app-link">Customers</a></li>
                    <li id="menu-item-46" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-46"><a href="https://www.digitalpigeon.com/plans/?utm_source=app&utm_medium=app-link">Plans</a></li>
                    <li id="menu-item-1889" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1889"><a href="https://www.digitalpigeon.com/blog/?utm_source=app&utm_medium=app-link">Blog</a></li>
                    <li id="menu-item-321" class="hidden-lg hidden-md menu-item menu-item-type-custom menu-item-object-custom current-menu-item menu-item-321"><a href="/trial">Start Free Trial</a></li>
                    <li id="menu-item-320" class="hidden-lg hidden-md menu-item menu-item-type-custom menu-item-object-custom menu-item-320"><a href="https://digitalpigeon.com/login">Login</a></li>
                </ul>
            </div>
        </div>
    </div>
</header>
<div class="content_wrap container">
    <div id="content">
         <div class="cssload-tetrominos">
            <div class="cssload-tetromino cssload-box1"></div>
            <div class="cssload-tetromino cssload-box2"></div>
            <div class="cssload-tetromino cssload-box3"></div>
            <div class="cssload-tetromino cssload-box4"></div>
        </div>
    </div>
    <div class="clearfix"></div>
</div>

<script type="text/javascript">
    window.baseCdnUrl = 'https://cdn.digitalpigeon.com/digitalpigeon/assets/v/60bb8ff';

    window._modules = {
        'app/externalMessage': { config: { messages: { heading: "404.heading", body: "404.body" } } },
        'app/AnalyticsManager': { config: { gtmAccount: "GTM-MWV99Q", env: '' }}
    };
</script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
<script src="https://cdn.digitalpigeon.com/digitalpigeon/assets/v/60bb8ff/assets/js/vendor.entry.js"></script>
<script src="https://cdn.digitalpigeon.com/digitalpigeon/assets/v/60bb8ff/assets/js/externalMessage.entry.js"></script>

<script type="text/javascript">
/* <![CDATA[ */
var shiftnav_data = {"shift_body":"off","shift_body_wrapper":"","lock_body":"on","lock_body_x":"off","swipe_close":"off","swipe_open":"off","swipe_tolerance_x":"150","swipe_tolerance_y":"60","swipe_edge_proximity":"80","open_current":"off","collapse_accordions":"off","scroll_panel":"on","breakpoint":"","touch_off_close":"on","scroll_offset":"100","disable_transforms":"off"};
/* ]]> */
</script>
<div class="shiftnav shiftnav-shiftnav-main shiftnav-right-edge shiftnav-skin-none shiftnav-transition-standard" id="shiftnav-main" data-shiftnav-id="shiftnav-main" style="max-height: 736px;">
		<div class="shiftnav-inner">
		<nav class="shiftnav-nav"><ul id="menu-main-menu-1" class="shiftnav-menu shiftnav-targets-default shiftnav-targets-text-default shiftnav-targets-icon-default"><li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-49 shiftnav-depth-0"><a class="shiftnav-target" href="https://www.digitalpigeon.com/?utm_source=app&utm_medium=app-link">Home</a></li><li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-625 shiftnav-depth-0"><a class="shiftnav-target" href="https://www.digitalpigeon.com/features/send-large-files/">Features</a></li><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-47 shiftnav-depth-0"><a class="shiftnav-target" href="https://www.digitalpigeon.com/customers/">Customers</a></li><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-46 shiftnav-depth-0"><a class="shiftnav-target" href="https://www.digitalpigeon.com/plans/">Plans</a></li><li class="hidden-lg hidden-md menu-item menu-item-type-custom menu-item-object-custom current-menu-item menu-item-321 shiftnav-depth-0"><a class="shiftnav-target" href="/trial">Start Free Trial</a></li><li class="hidden-lg hidden-md menu-item menu-item-type-custom menu-item-object-custom menu-item-320 shiftnav-depth-0"><a class="shiftnav-target" href="https://digitalpigeon.com/login">Login</a></li></ul></nav>		</div><!-- /.shiftnav-inner -->
	</div>
<script type="text/javascript" src="https://www.digitalpigeon.com/wp-content/themes/digitalpigeon/library/js/libs/superfish/js/superfish.js"></script>
<script type="text/javascript" src="https://www.digitalpigeon.com/wp-content/plugins/shiftnav-responsive-mobile-menu/assets/js/shiftnav.min.js"></script>
</body>
</html>
```

Raising with their support

**Comment by macmule** [comment]
```
May 19, 2021, 8:24 GMT+10

Hi Ben,

Thank you for the report the team is currently looking into the issue and hopes to release a fix as soon as possible.

In the meantime you can get the latest versions of the application on our "Digital Pigeon Apps" page.

Regards,
```

**Comment by macmule** [comment]
Ok now:

```
curl -L https://appcast.digitalpigeon.com/mac/v4/updates.xml
<rss xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle" xmlns:dc="http://purl.org/dc/elements/1.1/" version="2.0">
    <channel>
        <title>Digital Pigeon App Change Log</title>
        <link>https://appcast.digitalpigeon.com/mac/v4.1/updates.xml</link>
        <description>Most recent changes with links to updates.</description>
        <language>en</language>
        <item>
            <title>Version 4.2.14</title>
            <sparkle:minimumSystemVersion>10.10</sparkle:minimumSystemVersion>
            <description><![CDATA[
                <style type="text/css">
                * { font-family: sans-serif; }
                h2 { font-size: 16px; }
                li { font-size: 12px; }
                </style>
                <h2>Performance improvements and Bug fixes - 4.2.14</h2>
                <ul>
                <li>Updated Chromium engine for stability, performance and security fixes.</li>
                <li>Improved the performance of the app startup process.</li>
                <li>Significant performance improvements for uploads with lots of files addressing issues where the app would appear to 'lock up' for short periods of time.</li>
                </ul>
            ]]>
            </description>
            <pubDate>Friday, 15 November 2019 8:30:00 +1100</pubDate>
            <enclosure url="https://appcast.digitalpigeon.com/mac/v4.2/digital-pigeon-4.2.14.dmg"
            sparkle:version="0402014" sparkle:shortVersionString="4.2.14"
            length="76603880"
            type="application/x-apple-diskimage" />
        </item>
    </channel>
</rss>
```
